docs/03-01-system-requirements.md

```diff
@@ -26,12 +26,9 @@
         * session.auto_start = Off
         * date.timezone should be configured
 * MySQL 5.x (some work on sqlite has been done but is not supported)
-* Node.js and NPM with these packages installed globally:
-    * bower
-    * gulp
+* Node.js (7.7) and NPM with these packages installed globally:
     * uglify-js
     * uglifycss
-* Ruby, RubyGems and the bundler gem
```

Comment:
Only if you want to build the hologram style guide of the frontend styling, so not really required for the bundles I guess